int32 gain
---
bool result
